scripts/globals/additional_effects.lua

```diff
@@ -130,6 +130,7 @@ xi.additionalEffect.attack = function(attacker, defender, baseAttackDamage, item
         ABSORB_STATUS = 11,
         SELF_BUFF     = 12,
         DEATH         = 13,
+        LOCAL_VAR     = 14, -- arbitrary logic to apply a local var to the defender
```

Comment:
Summoning @TeoTwawki for AE changes, I don't know how they work or if this is OK etc.

Reply:
This was a pattern we decided was bad previously on Brigandish Blade and on the 4 winds for Gods. Better to do via a listener instead, even if that new listener itself connects with additional effect and not simply on hit I think that's the way to go for these things.

Reply:
can you point to an example of using a listener for this? Would it be something like adding a ranged attack listener if the player equips a stygian ash?

Reply:
@MowFord its going to need a whole new one, not an existing one. Not sure if we'd want it on the mob or on the item, either way one will need to check for the other. since there are technically multiple ixions (diff ID per zone) but only one item, probably on the mob checking that this was the item that hit it rather than the item having the item check if it was hitting "an ixion".

Reply:
for clarity, we only need to set the localvar to claim the mob, so seems like it would make more sense to do it for the person wielding the (consumable) stygian ash.

Looks like chaosbringer has an example of adding a listener on equip, maybe that's a fine method?

Reply:
oh, I was thinking "hit by ash" = set claim. multi taskinga t my dayjob atm

Reply:
Was not as simple as I hoped, but I got it done. The standard order of listeners is to run the state end after at other logic, but doing that meant it was happening after the ash was consumed/unequipped.

All working now